### ISSUE TITLE
Passing arguments and rendering wraps them in... one extra double quote in html?

### ISSUE THREAD

**Issue by y0nei** [jpsca/jinjax#46]
I have observed a weird behavior.

```jinja
<BaseLayout request={request}>
  <Card id="something" />
</BaseLayout>
```

`Card.jinja`:
```jinja
{#def type="info", hidden=false #}

<div {{ attrs.render() }} class="{{ 'hidden' if hidden }}">
   ...
</div>
```

The result is this property in the html element:
```html
id='"webgl-disabled-notice"'
```

No idea why. `jinjax = "^0.31"` 
How i render templates:
```py
from fastapi.templating import Jinja2Templates
from fastapi.responses import HTMLResponse
from jinjax.catalog import Catalog

_TEMPLATES_DIR = "src/templates"

templates = Jinja2Templates(
    directory=_TEMPLATES_DIR,
    # Whitespace control
    lstrip_blocks=True,
    trim_blocks=True
)

catalog = Catalog(jinja_env=templates.env)
catalog.add_folder(_TEMPLATES_DIR)
catalog.add_folder(f"{_TEMPLATES_DIR}/components")

def render(*args, **kwargs):
    return HTMLResponse(catalog.render(*args, **kwargs))
```

**Comment by shayne** [comment]
I'm still experiencing this with 0.34.

Example usage:
```html
<CheckboxItem
    class="border border-red-500"
    ...
 ```
 
Component:
 ```html
 {#def id, title, description #}

<div {{ attrs.render() }} class="relative flex items-start">
```

Results in:
```html
<div class="relative flex items-start" border-red-500&#34;="">
```

**Comment by jpsca** [comment]
Did you meant `<div {{ attrs.render(class="relative flex items-start") }}>`? Otherwise you would end with:
`<div class="border border-red-500"  class="relative flex items-start">`

In any case, this bug seems to be related to activating autoescape in Jinja. I'm working on a fix

**Comment by jpsca** [comment]
It should be fixed now, in the latest release (0.35) I just made
